UM/Settings/Profile.py

```diff
@@ -30,7 +33,13 @@ def __init__(self, machine_manager, read_only = False):
         super().__init__()
         self._machine_manager = machine_manager
         self._changed_settings = {}
-        self._name = "Unknown Profile"
+        self._changed_settings_defaults = {}
```

Comment:
Why does the profile now need to carry a copy of the default values? Is it a copy, actually?